Sources/Profile-View.php

```diff
@@ -267,70 +271,128 @@ function fetch_alerts($memID, $all = false, $counter = 0, $pagination = array(),
 
 		if (!empty($row['sender_id']))
 			$senders[] = $row['sender_id'];
+
+		if ($row['content_type'] == 'profile')
+		{
+			$members[] = $row['content_id'];
```

Comment:
might want to initialize **$members** as an array to avoid any cases where it wouldn't reach this point

Reply:
Good idea.